### PR TITLE
v1.10: NEWS and VERSION updates

### PR DESCRIPTION
**Pull request by jsquyres**
@rhc54 Please review

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -82,13 +82,13 @@ date="Unreleased developer copy"
...
+libmpi_so_version=12:7:0
 libmpi_cxx_so_version=2:3:1
-libmpi_mpifh_so_version=12:1:1
-libmpi_usempi_tkr_so_version=6:0:1
+libmpi_mpifh_so_version=12:1:0
+libmpi_usempi_tkr_so_version=7:0:2
 libmpi_usempi_ignore_tkr_so_version=8:0:2
 libmpi_usempif08_so_version=12:1:1
-libopen_rte_so_version=12:5:0
+libopen_rte_so_version=12:4:0
```
Comment (rhc54) [inline VERSION]: @jsquyres There was a change made to the ORTE library - why did you revert the change to the revision number?
```
# - If the library source code has changed at all since the last
#   update, then increment revision (`c:r:a' becomes `c:r+1:a').
```

Comment (jsquyres) [inline VERSION]: I only see the change to the rsh module (72459a77505b019fc0d96e11ed9c86fee1d832a1).  Was there another that I missed?

Comment (rhc54) [inline VERSION]: That was it - but it still represents a change in the library and a change in behavior of that component

Comment (rhc54) [inline VERSION]: or are you viewing the "library" in this case to be only the orte base code?

Comment (jsquyres) [inline VERSION]: Yes, the `.so` version numbers only reflect what happens in `libopen-rte.so` proper, for non-`disable-dlopen` builds.  Hence, changes to plugins are not reflected in these version numbers.
